app/assets/javascripts/modules/copy_clipboard.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * This code is used by the copy to clipboard function for the code snippet modal
+*/
+const setupCopyToClipboardListener = function () {
```

Comment:
This PR in GeoBlacklight adds a clipboard stimulus controller: https://github.com/geoblacklight/geoblacklight/pull/1441/files

It may be helpful to use it in the future.

Reply:
Thanks @marlo-longley ! I can look at changing this PR to the latest GBL EarthWorks branch and integrating that, if that makes more sense.